js/client/modules/@arangodb/testsuites/replication.js

```diff
@@ -72,6 +74,21 @@ function shellReplication (options) {
   return tu.performTests(opts, testCases, 'shell_replication', tu.runThere);
 }
 
+// //////////////////////////////////////////////////////////////////////////////
+// / @brief TEST: shell_replication
+// //////////////////////////////////////////////////////////////////////////////
+
+function shellClientReplication (options) {
+  let testCases = tu.scanTestPaths(testPaths.http_replication, options);
+
+  var opts = {
+    'replication': true
+  };
+  _.defaults(opts, options);
+
+  return tu.performTests(opts, testCases, 'replication_api', tu.runInLocalArangosh);
```

Comment:
It is a bit confusing that we use 3 different names here:
* the test suite is named `http_replication`. We probably don't want to rename this for downwards-compatibility reasons
* the function is named `shellClientReplicatation`
* it runs the server under the name `replication_api`.
:exploding_head: